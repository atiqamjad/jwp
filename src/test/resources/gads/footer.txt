<p>Any number of things can happen. Here are just a couple of things you have to consider. First off, I would check to see what all the analysts are saying about any stock you are about to try this on. Make sure the company has a solid dividend history. I would also caution against making the play on a stock that is due to report earnings while you are in the options period. Also keep in mind that as a general rule a stock will dip in direct relationship to the divided paid.</p>
<p>Obviously this strategy is not always going to play out as our hypothetical trade did. However, I have had results similar to that as well as some much better, and &#8220;yes&#8221; some that did not work at all. What makes the play less risky than the stand alone buy and hold trade is that no matter what the stock does, you get the dividend and the options premium money giving you that much downside protection on a move against you.</p>
<p>I had a number of stocks that I would hold in my account and merely roll over the option money and collect the dividend on a regular bases, double-dippers, and was very happy not to have the stock called away.</p>
<p>I was very fortunate that I had met a broker who became one of my best friends and taught me this method of investing. I strongly suggest that you seek the advice of a professional broker; money manager; your attorney; your accountant; your present, past or future wife or husband; your doctor; your heirs, your auto mechanic or anyone else in the world that you can think of before you try this or any method of investing. (Okay, I think that covers about everyone.)</p>
<p>To learn more about Covered Call writing, check the resources at http://www.TraderAide.com. Good luck and happy trading!</p>
<p>No permission is needed to reproduce an unedited copy of this article as long the About The Author tag is left in tact and hot links included. We do request that we be informed of where it is posted so reciprocal links can be considered. Email floyd@sbmag.org.</p>
<p>Floyd Snyder has been trading and investing in the stock market for three decades. He was on the forefront of the day trading craze that swept the nation back in the late1990&#8242;s both as a trader and as the moderator of one of the Internet&#8217;s largest real time trading rooms. He is the owner of <a target="_new" href="http://www.TraderAide.com">http://www.TraderAide.com</a>, Strictly Business Magazine at <a target="_new" href="http://www.sbmag.org">http://www.sbmag.org</a>, <a target="_new" href="http://www.FrameHouseGallery.com">http://www.FrameHouseGallery.com</a>and <a target="_new" href="http://www.EducationResourcesNetwork.com">http://www.EducationResourcesNetwork.com</a></p>