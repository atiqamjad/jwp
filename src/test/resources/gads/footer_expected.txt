Any number of things can happen. Here are just a couple of things you have to consider. First off, I would check to see what all the analysts are saying about any stock you are about to try this on. Make sure the company has a solid dividend history. I would also caution against making the play on a stock that is due to report earnings while you are in the options period. Also keep in mind that as a general rule a stock will dip in direct relationship to the divided paid.
Obviously this strategy is not always going to play out as our hypothetical trade did. However, I have had results similar to that as well as some much better, and &#8220;yes&#8221; some that did not work at all. What makes the play less risky than the stand alone buy and hold trade is that no matter what the stock does, you get the dividend and the options premium money giving you that much downside protection on a move against you.
I had a number of stocks that I would hold in my account and merely roll over the option money and collect the dividend on a regular bases, double-dippers, and was very happy not to have the stock called away.
I was very fortunate that I had met a broker who became one of my best friends and taught me this method of investing. I strongly suggest that you seek the advice of a professional broker; money manager; your attorney; your accountant; your present, past or future wife or husband; your doctor; your heirs, your auto mechanic or anyone else in the world that you can think of before you try this or any method of investing. (Okay, I think that covers about everyone.)
To learn more about Covered Call writing, check the resources at http://www.TraderAide.com. Good luck and happy trading!